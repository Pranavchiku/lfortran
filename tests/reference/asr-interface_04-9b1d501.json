{
    "basename": "asr-interface_04-9b1d501",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/interface_04.f90",
    "infile_hash": "14368b65a65dffde9d8d8ed4bc45203d43a32e68739474d9f18fff8b",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-interface_04-9b1d501.stdout",
    "stdout_hash": "6ca7b78b9fd1066652da4f1656c0b7fbbab0e66bc17d5b38fc4e0f90",
    "stderr": "asr-interface_04-9b1d501.stderr",
    "stderr_hash": "285bd753f0141253785af791a10ae09b25f51564130de90b88db6b76",
    "returncode": 0
}