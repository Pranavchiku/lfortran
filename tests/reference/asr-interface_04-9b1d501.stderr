warning: Implied save feature
  --> tests/../integration_tests/interface_04.f90:11:5
   |
11 |     integer, intent(inout) :: a
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/interface_04.f90:25:1
   |
25 | integer :: i
   | ^^^^^^^^^^^^ Assuming save attribute implicitly for variables
